src/components/content/deployedServices/myServices/MyServices.tsx

```diff
@@ -748,6 +750,21 @@ function MyServices(): React.JSX.Element {
                     </Button>
                 ),
             },
+            {
+                key: 'service actions',
```

Comment:
better to all it just 'actions'. 

Also for 'service configuration', pls rename to 'configuration'